packages/react/src/primitives/PhoneNumberField/__tests__/PhoneNumberField.test.tsx

```diff
@@ -0,0 +1,128 @@
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+
+import { PhoneNumberField } from '../PhoneNumberField';
+import { PhoneNumberFieldProps } from '../../types';
+import { ComponentClassNames } from '../../shared/constants';
+
+describe('PhoneNumberField primitive', () => {
+  const setup = async ({
+    defaultCountryCode = '+1',
+    label = 'Phone Number',
+    ...rest
+  }: Partial<PhoneNumberFieldProps>) => {
```

Comment:
Why make all these props optional?

Reply:
I made the props optional so that I could lean on default parameters here instead of passing `defaultCountryCode` and `label` to every `setup` function invocation. I'm open to changing it if you think it makes this setup function unclear for the tests.